hw/top_earlgrey/data/clocks.xdc

```diff
@@ -25,7 +25,7 @@ create_generated_clock -name clk_io_div4 -source [get_pin ${u_pll}/CLKOUT0] -div
 # therefore, Vivado propagates both clock edges down the clock network.
 # this implementation is not ideal - but we can at least tell Vivado to only honour the rising edge for
 # timing analysis.
-set_clock_sense -positive top_*/u_clkmgr_aon/u_no_scan_io_div2_div/gen_div2.u_div2/gen_fpga_buf.bufg_i_i_1__0/O
+set_clock_sense -positive top_*/u_clkmgr_aon/u_no_scan_io_div2_div/gen_div2.u_div2/gen_fpga_buf.bufg_i_i_1__*/O
```

Comment:
Oh wow, this is smart! Switched to your suggestion now.

In case Tim's PR is merged first, we can drop this one.

Reply:
Thanks for your approval @a-will . I needed to slightly tune the `get_pins` command to make it work.

Reply:
Ah, right, because that command was for Tim's PR, which introduces a choice of a BUFG or BUFR for various contexts (so the BUFG is under another level of hierarchy).